index.js

```diff
@@ -33,11 +34,39 @@ Toolkit.run(async tools => {
     body: env.renderString(body, templateVariables),
     title: env.renderString(attributes.title, templateVariables)
   }
-
   tools.log.debug('Templates compiled', templated)
-  tools.log.info(`Creating new issue ${templated.title}`)
+
+  if (updateExisting) {
+    let existingIssue
+    tools.log.info(`Fetching issues with title "${templated.title}"`)
+    try {
+      const existingIssues = await tools.github.search.issuesAndPullRequests({
+        q: `is:open is:issue repo:${process.env.GITHUB_REPOSITORY} in:title ${attributes.title}`,
```

Comment:
I believe we want to use `templated.title` here, since thats post-"templating" and will include things like variables and dates in the title. If your issue is supposed to have variables in the title, we don't want them to be considered duplicate.

```suggestion
        q: `is:open is:issue repo:${process.env.GITHUB_REPOSITORY} in:title ${templated.title}`,
```